ImageLounge/src/DkGui/DkWidgets.cpp

```diff
@@ -531,6 +541,7 @@ void DkExplorer::setCurrentPath(const QString &filePath)
     if (QFileInfo(filePath).isDir())
         mFileTree->expand(mSortModel->mapFromSource(mFileModel->index(filePath)));
 
+    mCurrentFilePath = new QString(filePath);

```

Comment:
```suggestion
    mCurrentFilePath = filePath;
```
Did you notice that this does not have its corresponding `delete`? This is why using `QString` directly is less error-prone.